lib/src/app_auth.dart

```diff
@@ -108,14 +108,20 @@ class AppAuth {
 
       return tokens["accessToken"];
     } on PlatformException catch (e) {
-      if (e.message?.toLowerCase()?.contains("network") == true) {
-        throw e.message; // probably bad network
-      }
 
-      if (e.code.startsWith("timeout")) {
+      if (e.code.startsWith("timeout_")) {
```

Comment:
In iOS side the Timeout code is not specified: just "timeout" - is it needed to specify the timeouts by scope, if we now handle the message and details?

Reply:
Chnanged swift part to 'timeout_' - code

Reply:
Might be a nice idea to do anyhow, might prove to be easier to understand where an error came from like that :)